modules/logging/filters.go

```diff
@@ -185,15 +187,113 @@ func (m IPMaskFilter) Filter(in zapcore.Field) zapcore.Field {
 	return in
 }
 
+type ActionType string
+
+const (
+	ReplaceType ActionType = "replace"
+	DeleteType  ActionType = "delete"
+)
+
+type cookieFilterAction struct {
+	Type  ActionType `json:"type"`
+	Name  string     `json:"name"`
+	Value string     `json:"value,omitempty"`
+}
+
+// CookiesFilter is a Caddy log field filter that
+// filters cookies.
```

Comment:
Same as the other PR, could use a usage explanation for JSON users, at least mention the action types.